kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/exercises/homework/FlightRepository.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.collections.adv.exercises.homework;
+
+import java.util.*;
+
+public class FlightRepository {
+
+    private static final Map<String, Set<Flight>> globalFlightsFrom = new HashMap<>();
+    private static final Map<String, Set<Flight>> globalFlightsTo = new HashMap<>();
+
+    public void addFlight(Flight flight) {
```

Comment:
Usun te metode, dodaj pole List<Flight> flights i konstruktor, ktory to pole ustawi.
W metodzie addFlightsFrom utworz mape Map<String, Flight> z pola flights (jak w przykladzie)